pyerrors/correlators.py

```diff
@@ -1244,37 +1241,37 @@ def __truediv__(self, y):
         else:
             raise TypeError('Corr / wrong type')
 
-    def __neg__(self):
+    def __neg__(self) -> "Corr":
         newcontent = [None if _check_for_none(self, item) else -1. * item for item in self.content]
         return Corr(newcontent, prange=self.prange)
 
-    def __sub__(self, y):
+    def __sub__(self, y: Union[Corr, float, ndarray, int]) -> "Corr":
```

Comment:
incomplete list for `y`.

Reply:
Okay, I stop here for the rest of the file. There are a few more instances below, where we have to adjust `y.